.github/workflows/ci.yml

```diff
@@ -23,6 +23,7 @@ jobs:
       - run: ./zig/download.sh
       - run: ./zig/zig fmt --check .
       - run: ./zig/zig build check
+      - run: ./zig/zig build scripts -- ci --build-docs
       - run: ./.github/ci/docs_check.sh
```

Comment:
Would be so good to move these checks to Zig as well! The link-checker in particular should be easy I think!

Reply:
I could add link checking to `file_checker.zig`. Then for all html files check the href attributes if these files exist.

Reply:
Yeah, that's exactly what I had in mind! Should be a follow up PR though!